modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/helpers/header.js

```diff
@@ -17,9 +17,42 @@
  *
  */
 
-var resources = function () {
-    return {
-        css:['common.css','sidepanel.css', 'left-navigation.css', 'font-wso2.css', 'bootstrap-dialog.min.css', 'typeahead.css'],
-        js:['sidepanel.js','greg-subscriptions-api.js','bootstrap-dialog.min.js', 'typeahead.bundle.min.js', 'typeahead.js']
+var name;
+var hps = require('/themes/store/helpers/header.js');
+var that = this;
+/*
+ In order to inherit all variables in the default helper
+ */
+for (name in hps) {
+    if (hps.hasOwnProperty(name)) {
+        that[name] = hps[name];
+    }
+}
+var fn = that.resources||function() { return {} };
+var resources = function(page, meta) {
+    var o = fn(page, meta);
+    if (!o.css) {
+        o.css = [];
+    }
+    if(!o.js){
+        o.js = [];
+    }
+    if(!o.code){
```

Comment:
Thank you Sameera for mentioning it,
This has been fixed with https://github.com/wso2/carbon-store/pull/834